### PR TITLE
Release script

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -137,33 +140,42 @@ publish-docs:
...
+update-hc:
+	make HOLO_REV=$(HOLO_REV) update-hc-sha
+	git checkout -b update-hc-$(HOLO_REV)
+	git add nixpkgs.nix
+	git commit -m hpos-rev:$(HOLO_REV)
+	git push origin HEAD
+
+update-hc-sha:
+	@if [ $(HOLO_REV) ]; then\
+		echo "⚙️  Updating holo-envoy using holochain rev: $(HOLO_REV)";\
```
Comment (timotree3) [inline Makefile]:
```suggestion
		echo "⚙️  Updating holo-envoy using holo-nixpkgs rev: $(HOLO_REV)";\
```

**Review by timotree3 — APPROVED** [review]
One small correction. Otherwise looks good to me!

File: Makefile [hunk 2]
```diff
@@ -137,33 +140,65 @@ publish-docs:
...
+	git checkout -b update-hc-$(HC_REV)
+	git add nixpkgs.nix
+	git commit -m hc-rev:$(HC_REV)
+	git push origin HEAD
+
+update-dnas:
+	@if [ $(DNA_VERSION) ]; then\
+		sed -i "24s/.*/  curl 'https:\/\/holo-host.github.io\/holo-hosting-app-rsm\/releases\/downloads\/$(shell echo $(DNA_VERSION) | tr .- _)\/core-app.$(shell echo $(DNA_VERSION) | tr .- _).happ' -o dnas\/holo-hosting-app.happ/" Makefile;\
+		sed -i "26s/.*/  curl 'https:\/\/holo-host.github.io\/servicelogger-rsm\/releases\/downloads\/$(shell echo $(DNA_VERSION) | tr .- _)\/servicelogger.$(shell echo $(DNA_VERSION) | tr .- _).happ' -o dnas\/servicelogger.happ/" Makefile;\
+		sed -i "28s/.*/  curl -LJ 'https:\/\/github.com\/Holo-Host\/dummy-dna\/releases\/download\/v$(DNA_VERSION)\/test.happ' -o dnas\/test.happ/" Makefile;\
```
Comment (timotree3) [inline Makefile:177]: Looks like this new DNA code didn't get reviewed. This sed code is using fixed line numbers right? That seems concerning.

Comment (zo-el) [reply Makefile:177]: We can replace that with a match script

**Pull request merged**